content/blog/2020/08/camel-quarkus-release-1.0.0/index.md

```diff
@@ -0,0 +1,131 @@
+---
+title: "Camel Quarkus 1.0.0 Released"
+date: 2020-08-10
+authors: ["ppalaga"]
+categories: ["Releases", "Camel Quarkus"]
+preview: What's new in Camel Quarkus 1.0.0
+---
+
+<sub><sup>Original image by <a href="https://commons.wikimedia.org/wiki/User:99of9">Toby Hudson</a> <a href="https://creativecommons.org/licenses/by-sa/3.0">CC BY-SA 3.0</a> via <a href="https://en.wikipedia.org/wiki/Camel_racing#/media/File:CamelRacingCamelCup2009Heat.JPG">Wikipedia</a></sup></sub>
+
+The Apache Camel community celebrates the release 1.0.0 of Camel Quarkus!
+
+## What is Camel Quarkus?
+
+Camel Quarkus brings the outstanding integration capabilities of Apache Camel to [Quarkus](https://quarkus.io/) - the
+toolkit for writing subatomically small and supersonically fast Java, Kotlin and Scala applications. In addition to
```

Comment:
I am not so sure Quarkus is well known for fast Scala applications. I dont think the Scala community think of Quarkus as a Scala framework they are using. I dont think you can find Scala mentioned on the quarkus website.

I would remove Scala.

Reply:
OK, let's remove Scala. Although there is scala extension and scala should work, it is not especially promoted from the Quarkus side.